kratos/utilities/specifications_utilities.h

```diff
@@ -52,6 +52,26 @@ namespace Kratos
  */
 namespace SpecificationsUtilities
 {
+    /**
+     * @brief This enum defines a "hash" used to identify if implicit/explicit or static time integration is considered
+     */
+    enum class TimeIntegration
+    {
+        Static   = 0,
+        Implicit = 1,
+        Explicit = 2
+    };
+
+    /**
+     * @brief This enum defines a "hash" used to identify if Lagrangian/Eulerian or ALE framework is considered
+     */
+    enum class Framework
+    {
+        Lagrangian = 0,
+        Euler      = 1,
```

Comment:
```suggestion
        Eulerian      = 1,
```

Reply:
BTW, for now this is not used. Currently I use the string, this is just an idea to later remove if the @KratosMultiphysics/technical-committee says so. PS: enums are a pain in python